File: F_24_Lecture_4/app/src/main/java/com/example/f_24_lecture_4/MainActivity.java
package com.example.f_24_lecture_4;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;
import android.widget.Toast;

import androidx.activity.EdgeToEdge;
import androidx.appcompat.app.AppCompatActivity;
import androidx.core.graphics.Insets;
import androidx.core.view.ViewCompat;
import androidx.core.view.WindowInsetsCompat;

public class MainActivity extends AppCompatActivity {

    EditText txtName;
    Button btnClickMe ;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        btnClickMe = findViewById(R.id.btnClickMe);
        txtName = findViewById(R.id.txtName);

        btnClickMe.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Intent intent = new Intent(MainActivity.this, SecondActivity.class);
                String text = txtName.getText().toString();
                intent.putExtra("name", text);
                startActivity(intent);
                // Toast.makeText(MainActivity.this, "Button Clicked", Toast.LENGTH_SHORT).show();
            }
        });
        };
    }

File: AskLolaLingo/app/src/main/java/com/example/asklolalingo/MainActivity.kt
package com.example.asklolalingo

import android.os.Bundle
import android.widget.Button
import androidx.appcompat.app.AppCompatActivity
import android.content.Intent ;

class MainActivity : AppCompatActivity() {

    private lateinit var btnLogin: Button ;

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        btnLogin = findViewById(R.id.btnLogin);

        btnLogin.setOnClickListener {
            val intent = Intent(this, AskLolaChat::class.java);
            startActivity(intent);
        }




        }



    }